src/app/core/services/issue.service.ts

```diff
@@ -147,6 +144,10 @@ export class IssueService {
         map((response: GithubIssue) => {
           response.comments = issue.githubComments;
           return this.createIssueModel(response);
+        }),
+        catchError((err) => {
+          this.logger.error(err); // Log full details of error first
+          return throwError(err.response.data.message); // More readable error message
```

Comment:
In that case, I wonder if the displayed error message can be changed to more layman terms for users to understand such as 'Could not validate user' for example.

This could be a new issue or new PR if we want to improve readability of error messages since it isn't part of the changes we are making here.

Reply:
Suggested message `Unable to assign the selected user(s) to this issue. Possible reason: a selected user has not joined the GitHub organization nus-cs2103-AY2122S2 yet`
Yes, it is fine to do as a separate PR too.

Reply:
Thanks for the suggestion! I think I can implement the more layman error message in a separate PR from this. It'll will probably only be done after the finals week, or I can open this as an issue that beginners can take up.

Reply:
Yes, I think that's a good idea.